assets/src/edit-story/elements/media/edit.js

```diff
@@ -139,15 +140,15 @@ function MediaEdit({ element, box }) {
     <Element>
       {isImage && <FadedImage {...fadedMediaProps} src={resource.src} />}
       {isVideo && (
-        <FadedVideo {...fadedMediaProps}>
+        <FadedVideo {...fadedMediaProps} loop={loop}>
```

Comment:
Not sure if looping should be applied to edit mode at all. Could confirm with UX but seems that perhaps not, especially since there are two videos on top of each other in edit mode. If these would be out of sync for some reason, could be quite odd. Thoughts?

Reply:
Happy to review for now. Removed in https://github.com/google/web-stories-wp/pull/653/commits/7e7ca0cb175e7c8dc7a699de71c26628267e379b